cpp/src/io/orc/stripe_data.cu

```diff
@@ -1732,8 +1739,9 @@ CUDF_KERNEL void __launch_bounds__(block_size)
         // Adjust the maximum number of values
         if (numvals == 0 && vals_skipped == 0) {
           numvals = s->top.data.max_vals;  // Just so that we don't hang if the stream is corrupted
+        } else {
```

Comment:
moved to the else branch to make it clear that there's no race condition between L1740 and L1743